tools/gpstest/src/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 OSQZSS
```

Comment:
gitignore this repo and add a script to clone it and do any other relevant setup

Reply:
here's an example of something like this: https://github.com/commaai/openpilot/blob/master/selfdrive/debug/profiling/palanteer/setup.sh